src/gc/gc.cpp

```diff
@@ -7417,7 +7414,7 @@ int gc_heap::grow_brick_card_tables (uint8_t* start,
 #endif
 
             //delete (uint32_t*)((uint8_t*)ct - sizeof(card_table_info));
-            if (!GCToOSInterface::VirtualRelease (mem, alloc_size_aligned))
+            if (!GCToOSInterface::VirtualRelease (mem, alloc_size))
```

Comment:
>alloc_size [](start = 55, length = 10)

this arg is actually not used, could you please remove it from the VirtualRelease call's signature?